pkg/pointerdb/audit/audit_segment.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+	"crypto/rand"
+	"math/big"
+	"sync"
+
+	"github.com/vivint/infectious"
+
+	"storj.io/storj/pkg/eestream"
+	"storj.io/storj/pkg/paths"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/pointerdb/pdbclient"
+	"storj.io/storj/pkg/storage/meta"
+)
+
+// Audit to audit segments
+type Audit struct {
+	pointers pdbclient.Client
+	lastPath *paths.Path
+	mutex    sync.Mutex
+}
+
+// NewAudit creates a new instance of audit
+func NewAudit(pointers pdbclient.Client) *Audit {
+	return &Audit{
+		pointers: pointers,
+	}
+}
+
+// Stripe is a struct that contains stripe info
+type Stripe struct {
+	stripe int
+}
+
+// NextStripe returns a random stripe to be audited
+func (a *Audit) NextStripe(ctx context.Context) (stripe *Stripe, more bool, err error) {
+	a.mutex.Lock()
+	defer a.mutex.Unlock()
+
+	// retrieve a random list of pointers
+	var pointerItems []pdbclient.ListItem
+	var path paths.Path
+
+	// need to get random limit
```

Comment:
You don't need a random limit. As we've discussed it's enough to pass `0` to the List method and rely on the internal server-side limit. So you can remove this comment.

Reply:
this was an oversight on the comment (it should have been deleted) i just missed it :)

Reply:
Let's remove it then :)

Reply:
fixed!